src/Files.App/Views/ContentLayouts/BaseGroupableLayout.cs

```diff
@@ -29,7 +29,7 @@ public abstract class StandardViewBase : BaseLayout
 
 		public ICommandManager Commands { get; } = Ioc.Default.GetRequiredService<ICommandManager>();
 
-		public StandardViewBase() : base()
+		public BaseGroupableLayout() : base()
```

Comment:
What does "Groupable" stand for?

Reply:
StandardViewBase hold groupable(Group By) control information; layouts, such as GridView, DetailsView. ColumnsViewLayout doesn't use this class because it cannot be grouped.

Reply:
Can you call it "GroupBy" instead of "Groupable"?

Reply:
Groupable doesn't suit you? I would feel odd if it was "BaseGroupByLayout" "Group by layout".

Reply:
We can keep as is but can you add a comment to the top of the class?

Reply:
Added

Reply:
Wait, should I add *Page* suffix to all of them? I suppose to do I think. @yaira2 What do you think?

Reply:
Your call